src/utils/githubApi.ts

```diff
@@ -103,9 +103,17 @@ export function getGitHubClient(token = ''): Octokit {
   if (!_GitHubClientCache[githubApiToken]) {
     const attrs: any = {
       auth: `token ${githubApiToken}`,
-      log: logger,
     };
 
+    // Silence debug logs, as they do not provide any useful information
+    // about the requests, yet they are very noisy and make it difficult
```

Comment:
🤣 So I guess this is why it was done this way in the past but now with the comment, it is much much clear and hopefully nobody will reinvent that wheel.